### PR TITLE
Set up Desktop version on Portafoilo project (html & css only)

### PR DESCRIPTION
**Pull request by chelobotix**
In this PR, I fulfilled the following tasks:
- [x] Add media query for the Desktop version (Desktop: 768px or wider)
- [x] Create Heading Section
- [x] Create My Works Sections
- [x] Create About me Section 
- [x] Create a Contact Form with an endpoint at [https://formspree.io/html](url)
- [x] Update README file with new features and new logo

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -226,150 +249,167 @@ <h4>Multi-Post Stories Gain+Glory</h4>
...
+              <li>
+                <a href="#" target="_blank"> <img src="images/angellist.svg" /></a>
+              </li>
+            </ul>
+          </form>
+        </div>
+      </section>
+    </div>
 
     <section class="email_footer">
```
Comment (huseyinbiyik) [inline index.html:417]: Fantastic job @chelobotix. But kindly separate the social media section with an hr or border. And there is too much white space after the social media section. Please be strict about the Figma template and fix it. 👨‍🎨

Your design:
Screenshot 2022-12-16 at 12 52 36 AM

Figma:
Screenshot 2022-12-16 at 12 52 51 AM

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi @chelobotix,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Almost there

## Highlights
- Very good implementation of the given design 👌
- Linter checks passing 🟢

## Required Changes ♻️
- Great job @chelobotix! But let's fix the overflowing issue occurring on all sizes. Kindly check the gif below.

Screen Recording 2022-12-16 at 12 58 01 AM

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [OPTIONAL] Please fix the license link on your Readme file. It is crucial for a professional-looking repository. 👨‍💼

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (chelobotix) [reply index.html:417]: @huseyinbiyik Thanks for r your feedback. Could you check if the overflowing issue has been resolved?
Thank you very much

Comment (huseyinbiyik) [reply index.html:417]: @chelobotix, they still exist. I checked on different browsers. There are still some issues.🙃

**Comment by chelobotix** [comment]
The following observations mentioned in the above comments have been fixed:
### Overflow issue
Peek 2022-12-16 10-23
image

### Separate the social media section with an hr or border
image

### Fix License broken link

image


Thank you

File: index.html [hunk 2]
```diff
@@ -226,150 +249,169 @@ <h4>Multi-Post Stories Gain+Glory</h4>
...
-            </li>
-          </ul>
-        </form>
-      </div>
-    </section>
+            <div class="divbtn_field">
+              <button class="btn-form btn_effects" type="submit">Get in touch</button>
+            </div>
+
+            <hr>
```
Comment (ichala) [inline index.html]: - [x] please note that the `<hr>` that you added as requested by the previous reviewer is not visible and its in the wrong place , kindly fix it 

image

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @chelobotix 
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Good Design  ✔️ 


 ## Required Changes ♻️


- [ ] **_🛑  Check The Comments Below_**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (chelobotix) [reply index.html]: Dear @ichala, Could you give me your feedback if the hr tag is now in the requested position?
Thank you very much
image
image

Comment (chelobotix) [reply index.html]: dear @ichala, Could you give me your feedback?
Thanks

File: style.css [hunk 3]
```diff
@@ -1,632 +1,1330 @@
...
+    letter-spacing: 0.03em;
+    border: 0;
+    cursor: pointer;
+  }
+
+  .hr_dsk {
+    width: 186px;
+    margin-left: 0;
+    margin-top: 170px;
+  }
```
Comment (levy002) [inline style.css]: - [x] Kindly make sure the line touches on both ends of you page in order to meet the provided Figma design as requested by the previous reviewers.

   - ### YOUR DESIGN
    image

    - ### FIGMA DESIGN
    
   image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @chelobotix   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 3 above)
Comment (chelobotix) [reply style.css]: Dear @levy002.
Kindly, could you check if this is correct now?
image
Thank you very much!

**Review by ha-manel — APPROVED** [review]
# STATUS: APPROVED ✅

## Hi @chelobotix 👋

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉**



## Highlights:
* Well-written PR description ✔️ 
* All tests passed ✔️
* Your website looks good 💯  


------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (ichala) [reply index.html]: It looks like you fixed the issue and ur project is approved 💯💯💯 Good job , keep rocking

**Pull request merged**